Sources/View/Abstract/BaseStackView.swift

```diff
@@ -51,10 +51,10 @@ class BaseStackView: BaseView {
         stackView.translatesAutoresizingMaskIntoConstraints = false
         
         // Constraints
-        let aTopConstraint = NSLayoutConstraint(item: stackView, attribute: .Top, relatedBy: .Equal, toItem: self, attribute: .Top, multiplier: 1.0, constant: 0.0)
-        let aLeftConstraint = NSLayoutConstraint(item: stackView, attribute: .Left, relatedBy: .Equal, toItem: self, attribute: .Left, multiplier: 1.0, constant: 0.0)
-        let aBottomConstraint = NSLayoutConstraint(item: stackView, attribute: .Bottom, relatedBy: .Equal, toItem: self, attribute: .Bottom, multiplier: 1.0, constant: 0.0)
-        let aRightConstraint = NSLayoutConstraint(item: stackView, attribute: .Right, relatedBy: .Equal, toItem: self, attribute: .Right, multiplier: 1.0, constant: 0.0)
+        let aTopConstraint = NSLayoutConstraint(item: stackView, attribute: .top, relatedBy: .equal, toItem: self, attribute: .top, multiplier: 1.0, constant: 0.0)
```

Comment:
Nice

Suggestion: 

- store constraint in array
- then use the not much publicized `NSLayoutConstraints.activateConstraints` method on that array (instead of setting `isActve` individually)